lib/tasks/rubocop.rake

```diff
@@ -18,5 +18,5 @@ RuboCop::RakeTask.new(:rubocop) do |task|
                     changed_files
                   end
 
-  task.fail_on_error = false
```

Comment:
`fail_on_error ` is by default `true` [1] so maybe @lebedev-yury (as original author of this task) can say if there is any reason why it was `false` in the first place.

[1] https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/rake_task.rb#L68